### DISCUSSION TITLE
Supporting rootless (non-root) connectors with v0.63.9

### DISCUSSION THREAD

**Discussion by nataliekwong** [airbytehq/airbyte#44924] [Platform Security Announcements]
# What 
As a security best practice, Airbyte has begun migrating our connectors to run rootless, or non-root versions.

Most organizations IT/Ops departments mandate non-root privileges for containers in their Kubernetes clusters. To ensure Airbyte aligns with those security policies, the Airbyte platform [v0.63.9](https://github.com/airbytehq/airbyte-platform/releases/tag/v0.63.9) was updated to run both root and non-root versions of connectors.

## Next Steps
If you have upgraded to Airbyte v0.63.9 or beyond, no action is required. 

If you have not yet upgraded to Airbyte or beyond, you are required to update the platform version of Airbyte. If you do not upgrade your platform version, your syncs will begin to fail as connectors are migrated to the non-root version.

We expect that all connectors will be migrated to their non-root version by the end of October 2024.

**Comment by lalitB-dev** [comment]
Hi ,
We are running Airbyte in an EC2 instance and have a few questions regarding the above changes.
What does it mean when it's stated that "Airbyte is migrating connectors to run as rootless"?
Do we need to check the user with respect to the containers which are created dynamically during a sync for the source/destination connectors ?
